languages/tgmpa-de_DE.po

```diff
@@ -1,63 +1,61 @@
+# Translation of TGM-Plugin-Activation in German (Germany)
+# This file is distributed under the same license as the TGMPA package.
 msgid ""
 msgstr ""
-"Project-Id-Version: TGM Plugin Activation\n"
-"POT-Creation-Date: 2016-01-21 17:25+0100\n"
-"PO-Revision-Date: 2016-01-22 13:49+0100\n"
+"Project-Id-Version: TGM-Plugin-Activation\n"
```

Comment:
So should there be hyphens in the project name or not ? Which is the correct way ?